webpack.config.js

```diff
@@ -19,10 +24,32 @@ var defineEnvPlugin = new webpack.DefinePlugin({
   __ABOUT_MAX_LENGTH__: JSON.stringify(process.env.ABOUT_MAX_LENGTH || null),
   __DEV__: isDev,
   __TEST__: false,
-  __DEV_TOOLS__: (process.env.DEV_TOOLS != null) ? process.env.DEV_TOOLS : (isDev ? true : false)
+  __DEV_TOOLS__: (process.env.DEV_TOOLS !== null) ? process.env.DEV_TOOLS : (isDev ? true : false)
 });
 
-var plugins = [ defineEnvPlugin, new ExtractTextPlugin('style.[contenthash].css') ];
+var plugins = [
+  defineEnvPlugin,
+  new ExtractTextPlugin('style.[contenthash].css'),
+  new CopyWebpackPlugin([
+    {
+      from: 'static',
+      transform: (content, path) => {
+        var code = fs.readFileSync(path, 'utf8');
+        var result = uglifyJS.minify(code);
+        return result.code;
+      }
+    }
+  ]),
+  new HtmlWebpackPlugin({
+    template: 'index.ejs',
+  }),
+  new HtmlWebpackIncludeAssetsPlugin({
+    assets: ['pdfkit.js', 'blob-stream.js'],
+    hash: true,
+    append: true,
```

Comment:
In this case, these assets just need to be in place by the time the web worker is invoked to create a print pdf, which is when a user navigates to the daily view.

Technically, it doesn't even need to be loaded by the main thread.  The only reason I have it being loaded there at all is for cache-busting, forcing the browser to reload it when we deploy with a new hash.

I recommend we keep the bundle.js loading first for performance.